### PR TITLE
Update SPL_Center.org.xml

### PR DESCRIPTION
**Pull request by I-have-50-open-PRs-to-HTTPS-EVERYWHERE**

### CODE REVIEW

File: src/chrome/content/rules/SPL_Center.org.xml [hunk 1]
```diff
@@ -22,12 +8,12 @@
 -->
 <ruleset name="SPL Center.org (partial)">
 
-	<!--target host="splcenter.org" />	Needs clearnet testing	-->
+	<target host="splcenter.org" />
+	<target host="www.splcenter.org" />
 	<target host="donate.splcenter.org" />
 	<!--target host="edit.splcenter.org" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/SPL_Center.org.xml]: WFM

File: src/chrome/content/rules/SPL_Center.org.xml [hunk 2]
```diff
@@ -22,12 +8,12 @@
...
 	<!--target host="secure.splcenter.org" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/SPL_Center.org.xml]: Does not exist anymore

File: src/chrome/content/rules/SPL_Center.org.xml [hunk 3]
```diff
@@ -22,12 +8,12 @@
...
 	<!--target host="selma.splcenter.org" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/SPL_Center.org.xml]: Please move this comment to the top and document the issue (mismatch).

File: src/chrome/content/rules/SPL_Center.org.xml [hunk 4]
```diff
@@ -22,12 +8,12 @@
 -->
 <ruleset name="SPL Center.org (partial)">
```
Comment (J0WI) [inline src/chrome/content/rules/SPL_Center.org.xml]: I think we can remove the `partial` flag now

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@J0WI Added.

**Pull request merged**